apps/dash-covid-xray/README.md

```diff
@@ -2,7 +2,10 @@
 
 ## About this app
 
-This is a demo of the Dash interactive Python framework developed by [Plotly](https://plot.ly/).
+This app shows how to explore 3-D data of chest tomography thanks to Dash. 
```

Comment:
```suggestion
This app shows how to explore 3-D chest tomography data using Dash. 
```
It's true that "thanks to" is more accurate because we're just having Dash help us explore the data, but it just in English we're less thankful ;).